pkg/apis/core/types.go

```diff
@@ -6016,3 +6025,15 @@ type PortStatus struct {
 	// +kubebuilder:validation:MaxLength=316
 	Error *string
 }
+
+// LoadBalancerIPMode represents the mode of the LoadBalancer ingress IP
+type LoadBalancerIPMode string
+
+const (
+	// LoadBalancerIPModeVIP indicates that the traffic passing through this LoadBalancer
+	// is delivered with the destination IP set to the specified LoadBalancer IP
+	LoadBalancerIPModeVIP LoadBalancerIPMode = "VIP"
+	// LoadBalancerIPModeProxy indicates that the specified LoadBalancer acts like a proxy,
+	// changing the destination IP to the node IP and the source IP to the LoadBalancer (mostly private) IP
+	LoadBalancerIPModeProxy LoadBalancerIPMode = "Proxy"
+)
```

Comment:
you don't have to redeclare these constants here, just use the one of staging/src/k8s.io/apis if you need them

Reply:
scratch it, it seems they are redeclared for other types